src/odemis/acq/test/scan_test.py

```
@@ -165,6 +166,9 @@ class TestVectorAcquisition(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
+        if sys.version_info < (3, 9):
+            raise SkipTest("semnidaq does not work for Ubuntu 20.04 or lower")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)